config/features.yml

```diff
@@ -1,107 +1,158 @@
 ---
 # Add a new feture toggle here to ensure that it is initialized in all environments.
-# Features are defaulted to enabled in development and test environments and disabled in all others.
+#
+# Features are enabled by default in development and test environments, and disabled by default in production.
+# Set a given environment to 'true' or 'false' in `default_enabled_environments` to override this behavior.
+#
 # The description should contain any relevant information for an admin who may toggle the feature.
 #
 # The actor_type should be either `user` for features you want to be "sticky" for a logged in user (default)
 #  or `cookie_id` of you wish to use the Google Analytics id as the unique identifier.
-
 features:
   dashboard_show_covid19_alert:
     actor_type: user
+    default_enabled_environments:
     description: >
       Allows for showing a special COVID19 alert on the My VA Dashboard
   facility_locator_show_community_cares:
     actor_type: user
+    default_enabled_environments:
+      development: true
```

Comment:
We are no longer using `facility_locator_show_community_cares`.

Can you please replace it with this?

```yml
  facilities_ppms_suppress_community_care:
    actor_type: user
    description: Hide ppms community care searches
    default_enabled_environments:
      development: false
```